resources/postinstall.js

```diff
@@ -0,0 +1,31 @@
+var osenv = require('osenv');
+var fs = require('fs-extra');
+var keypair = require('keypair');
+
+var dir = osenv.home() + '/.tessel';
+var filename = 'id_rsa';
+var filepath = dir + '/' + filename;
+
+// If keypair doesn't already exist.
+// TODO: fs.exists is going to be deprecated, what's a better way to do this?
```

Comment:
I think you mean `fs.existsSync` is going to be deprecated. You can replace it with

```
fs.exists(filename, callback(exists))
```